stride/chain.json

```diff
@@ -0,0 +1,114 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "stride",
+    "status": "live",
+    "network_type": "mainnet",
+    "updatelink": "https://raw.githubusercontent.com/osmosis-labs/osmosis/main/chain.schema.json",
+    "pretty_name": "Stride",
+    "chain_id": "stride-1",
+    "bech32_prefix": "st",
```

Comment:
the PR on slip173 is for prefix 'stride'. Is this 'st' meant to be 'stride'?

Reply:
yes, fixed. thanks!